### DISCUSSION TITLE
Allow Zeus to be in the loadout

### DISCUSSION THREAD

**Discussion by thanhhungtn1** [yonilerner/cs2-retakes-allocator#105] [Q&A] [answered]
I cannot find anywhere have config to give the players the zeus by default or maybe I missed it anywhere.
Please could you guide me how to do it. 
Thank you so much.

**Comment by yonilerner** [comment, marked as answer]
The next version will have a `ZeusPreference` config that you can set to `Always` if you want to give players zeus

**Reply by thanhhungtn1** [reply]
Thank man  that really help to make some fun in the game 👍
